docs/core/compatibility/sdk/8.0/runtimespecific-app-default.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Breaking change: Runtime-specific apps no longer self-contained"
+description: Learn about a breaking change in the .NET 8 SDK where apps that specify a runtime identifier are no longer self-contained by default.
+ms.date: 06/05/2023
+---
+# Runtime-specific apps no longer self-contained
+
+Runtime-specific apps, or .NET apps with a `RuntimeIdentifier`, are no longer [self-contained](../../../deploying/index.md#publish-self-contained) by default. Instead, they are [framework-dependent](../../../deploying/index.md#publish-framework-dependent) by default.
+
+This is a breaking change in the following situations:
+
+- If you deployed, distributed, or published your app and didn't explicitly add the `SelfContained` property, but also didn't require that the .NET runtime be installed on the machine for it to work. In this case, you may have relied on the previous behavior to produce a non-framework-dependent app by default.
+
+- If you rely on the IL Link tool. In this case, you'll have to take the steps described under [Recommended action](#recommended-action) to use IL Link again.
+
+  > [!NOTE]
+  > Some publish properties, like `PublishTrimmed`, `PublishSingleFile`, and `PublishAot`, currently require `SelfContained` to work. If you use these properties, you'll need to add the `SelfContained` property.
```

Comment:
We add SelfContained for all of these properties still after this change, though only when running `dotnet publish` *(`PublishSelfContained`  is set to true if these properties are and `PublishSelfContained` was not explicitly set to false. That may need to be clarified here.